flink-tests/src/test/java/org/apache/flink/test/runtime/ShuffleCompressionITCase.java

```diff
@@ -133,18 +136,19 @@ public LongValueSource(Environment environment) {
 		@Override
 		public void invoke() throws Exception {
 			ResultPartitionWriter resultPartitionWriter = getEnvironment().getWriter(0);
-			RecordWriterBuilder<LongValue> recordWriterBuilder = new RecordWriterBuilder<>();
+			RecordWriterBuilder<LongValue> recordWriterBuilder = new RecordWriterBuilder<>()
+				.setChannelSelector(new BroadcastPartitioner());
```

Comment:
It might be better to give two cases for covering both broadcast and non-broadcast

Reply:
I will supplement a test for non-broadcast partitioner soon.